server/test/support/CfTestHelpers.java

```diff
@@ -0,0 +1,39 @@
+package support;
+
+import static org.mockito.Mockito.mockStatic;
+
+import com.google.common.collect.ImmutableMap;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+import org.mockito.MockedStatic;
+import play.api.test.Helpers;
+
+public class CfTestHelpers {
```

Comment:
There's only 1 class/test that use Cf in file names and the rest use CiviForm.  I'm fine with well known abbreviations but we don't seem to necessarily have that with Cf or use it..  Would we enforce using Cf going forward to establish this?

Reply:
I can go either way with a very mild preference for `Cf` for brevity. Thoughts?